### DISCUSSION TITLE
Support for ConfigurableSamplerProvider

### DISCUSSION THREAD

**Discussion by suman-aella** [open-telemetry/opentelemetry-java-instrumentation#2807] [Q&A]
Is the support or functionality for "ConfigurableSamplerProvider" interface provided in the otel SDK v1.1 yet to be implemented in the java agent? 
[ConfigurableSamplerProvider](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/ConfigurableSamplerProvider.java
)
Will it be coming as part of later versions of the java agent (v 1.1 )??

**Comment by iNikem** [comment]
SDK autoconfiguration module and all its extensions and SPIs are already supported either via [custom distribution approach](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/examples/distro) or via `otel.javaagent.experimental.initializer.jar` configuration property. The latter is experimental, not stable yet, and thus is not documentation atm. See also #2442
